core/src/main/scala/kafka/consumer/TopicCount.scala

```diff
@@ -120,6 +120,10 @@ private[kafka] class StaticTopicCount(val consumerIdString: String,
     }
   }
 
+  override def hashCode(): Int = {
```

Comment:
It seems a bit odd that this class implements `equals` but the `WildcardTopicCount` one doesn't.